spec/presenters/two_factor_auth_code/phoneable.spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+phone_update_path = 'a/link/to/update'
+
+describe TwoFactorAuthCode::Phoneable do
+  let(:presenter) do
+    TwoFactorAuthCode::PhoneDeliveryPresenter.new(attributes_for(:generic_otp_presenter))
+  end
+
+  let(:unconfirmed) do
+    data = attributes_for(:generic_otp_presenter).merge!(
+      reenter_phone_number_path: phone_update_path,
+      unconfirmed_phone: true
+    )
+
+    TwoFactorAuthCode::PhoneDeliveryPresenter.new(data)
+  end
+
+  describe '#phone_fallback_link' do
+    it 'returns a link for voice delivery if user initially selected sms' do
+      output = presenter.phone_fallback_link
+
+      expect(output).to include(t('links.two_factor_authentication.voice'))
+      expect(output).to match(/<a href/)
+    end
+
+    it 'return a link for sms delivery if user initially selected voice' do
+      output = presenter.phone_fallback_link
+
+      expect(output).to include(t('links.two_factor_authentication.sms'))
+    end
```

Comment:
So this functionality is already tested in a view spec here: https://github.com/18F/identity-idp/blob/2c70e3fdaaef10388b13475a07b954609f6945d9/spec/views/two_factor_authentication/otp_verification/show.html.slim_spec.rb#L76

In this case the first part of the string isn't tested, however, I do test that the entire link is present. Given that this test implicitly calls the `phone_fallback_link` in the `Phoneable` module, this test would be redundant right? I'll still add a test that this module includes this method, but as you said, it's much easier to test html output in the view spec where we get all the nice rails helper methods.